src/mote/templates/mainpage.html

```diff
@@ -394,5 +395,34 @@ <h6 class="modal-title head fw-bold">Recent conversations</h6>
                 </div>
             </div>
         </div>
+        <div class="toast-container"> 
+            <div class="position-fixed bottom-0 end-0 p-3" style="z-index: 11">
+                <div id="liveToast" class="toast" role="alert" aria-live="assertive" aria-atomic="true">
+                    <div class="toast-body">
+                        A meeting has just ended
+                        <div class="mt-2 pt-2 border-top">
+                            <a id="toast-btn" type="button" class="btn btn-primary btn-sm" href="" target="_blank">Open meeting</a>
+                            <button type="button" class="btn btn-secondary btn-sm" data-bs-dismiss="toast">Close</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
     </body>
+    <script>
+      var socket = io.connect("http://" + document.domain + ":" + location.port)
```

Comment:
I don't think you need to pass any arguments if you're using the same server
```suggestion
      var socket = io.connect()
```